.env.example

```diff
@@ -0,0 +1 @@
+APPROVED_TOKENS=<Tokens seperated by comma>
```

Comment:
It is better for every file to have a blank line at the end

Reply:
Oh yes :D Lessons learned from this afternoon :point_up: :D

Reply:
🤓